app/src/main/java/com/innowisegroup/datetimepicker/MainActivity.kt

```diff
@@ -10,13 +10,14 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        val dateTimePickerDialog = DateTimePickerDialog(
-                null,
-                null,
-                null,
-                wrapSelectionWheel = false,
-                withOnlyDatePicker = false,
-                withOnlyTimePicker = false)
+        val dateTimePickerDialog = DateTimePickerDialog.newInstance(
+            initialLocalDateTime = null,
+            initialLocalTime = null,
+            minLocalDateTime = null,
+            maxLocalDateTime = null,
+            wrapSelectionWheel = false,
+            fragmentToCreate = FragmentToCreate.BOTH
```

Comment:
Such an API (null, null, null.....) is not what the developer wants to see when creating picker. Let's think how we can improve it and make more friendly to call.